feature/bgp/policybase/otg_tests/actions_med_localpref_prepend_flow_control/actions_MED_LocPref_prepend_flow_control_test.go

```diff
@@ -269,15 +248,15 @@ func configureASLocalPrefMEDPolicy(t *testing.T, dut *ondatra.DUTDevice, policyT
 		actions.PolicyResult = oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE
 	case setMEDPolicy:
 		if strings.Contains(policyValue, "+") {
-			if deviations.BgpSetMedUnionTypeUnsupported(dut) {
-				metric, _ := strconv.Atoi(policyValue)
-				gnmiClient := dut.RawAPIs().GNMI(t)
-				config := metricAdd(dut, setMEDPolicy, metric)
+			if deviations.BgpSetMedV7Unsupported(dut) {
 				t.Logf("Push the CLI config:%s", dut.Vendor())
-				gpbSetRequest := buildCliConfigRequest(config)
-				if _, err := gnmiClient.Set(context.Background(), gpbSetRequest); err != nil {
-					t.Fatalf("gnmiClient.Set() with unexpected error: %v", err)
+				metric, _ := strconv.Atoi(policyValue)
+				var config string
+				switch dut.Vendor() {
+				case ondatra.JUNIPER:
+					config = juniperBgpPolicyMEDAdd(setMEDPolicy, metric)
 				}
+				helpers.GnmiCLIConfig(t, dut, config)
```

Comment:
Add a default: case with a t.Fatal saying that the deviation BgpSetMedV7Unsupported for vendor %s is  required but no implementation is defined.

Reply:
Updated , please check